conf/local-config.json

```diff
@@ -1,2 +1,5 @@
 {
+  "enable_admin_logging": true,
```

Comment:
Should this be false too because extra setup of aws credential is needed to run locally?

Reply:
Good point, I only thought about it from my machine.